### ISSUE TITLE
Add a node list

### ISSUE THREAD

**Issue by normanrz** [scalableminds/webknossos#4792]
Currently, there is only a list of comments. All nodes that don't have a comment don't show up there.
On another note: It is not possible right now to find all the node comments that belong to a specific tree.

**Comment by philippotto** [comment]
Makes sense, however, we need to be careful with performance. Maybe this can be optional.

> On another note: It is not possible right now to find all the node comments that belong to a specific tree.

I think, this is currently possible? The comments are grouped by tree, which means you can scroll to the specific tree and see all node comments which belong to it?

![image.png](https://images.zenhubusercontent.com/573597a9bf93871ccd9af943/8db9a6e4-9ef9-47c4-810e-8f50b40b8b5e)

**Comment by hotzenklotz** [comment]
While talking with @normanrz and @philippotto these days about branch points I came across this once again. The current comments/(skeleton) UI/UX has several downsides:
- individual nodes are hidden from the user (there is no list)
  - node can only be selected with mouse click which means you need to be in the right plane
  - nodes can only be navigated through with shortcuts
- branch points are also not listed in the UI
  - it is very unclear how many branch points are in the invisible queue  (no list anywhere)
  - branch point navigation/marking/unmarking can only done through B/J shortcuts
- the comment tab is under used / mostly empty and uses screen real estate

How about we rename the `Comments` tab to `Nodes` and upgrade it a bit. It could list all nodes (virtualized) with sensible filter options (and good default) to make sense of it all:

Filters:
- only show nodes with a comment or branch point (default) 
- only show branch points
- only show nodes of current tree
- show all nodes

This would make interactions with nodes more approachable and easier, e.g. with right-click menu as for the skeletons. Also branch points would have a place in the interface. 

Implementation-wise, we can built this step by step, since we can just reuse most of the `Comments` tab Ui and build on it. I think this can be split into small work packages.  

Example, mock up:
![Untitled-2022-02-04-1426](https://user-images.githubusercontent.com/1105056/152539087-a976873c-b0ee-466a-ab37-5d136abd6f44.png)

The only downside with my mockup is the duplication of the tree/group structure from the `skeleton` tab. (But that is already the case)

**Comment by hotzenklotz** [comment]
(cc) @fm3 ^

**Comment by hotzenklotz** [comment]
Also related to https://github.com/scalableminds/webknossos/issues/4969

**Comment by philippotto** [comment]
As already said, I like the idea :+1: However, I would organize the hierarchy as follows:

```
Tree 1
  Comments
    Comment A
    Comment B
  Branchpoints
    Branchpoint 1
Tree 2
  Comments
    Comment C
```

So, everything is grouped by the tree it belongs to. Otherwise, the hierarchy would feel a bit weird to me.

**Comment by normanrz** [comment]
> So, everything is grouped by the tree it belongs to. Otherwise, the hierarchy would feel a bit weird to me.

I think that could be intuitive, but the ordering of branchpoints (ie. jump list) gets lost.
